packages/Textarea/src/Textarea.js

```diff
@@ -105,7 +108,11 @@ function Textarea(props) {
         data-pka-anchor="textarea"
         disabled={isDisabled}
         readOnly={isReadOnly}
-        ref={setRef}
+        onChange={handleChange}
+        ref={node => {
```

Comment:
Is this to address the comment on the previous PR?
Granted I'm not sure if anyone uses `inputRef` so may not be worth discussing. But note that this change actually does nothing. You used to have this function in a variable, now you have it inline. But point is, it's a new function each render anyway (which is the reason React will call it with `null`).
I think this can be partially made better by memoizing
```jsx
const setRef = React.useCallback(() => {
  ...
}, [props.inputRef]);
```
This will preserve referential integrity (unless `inputRef` changes) but honestly not sure it's worth it. I just did a search in the entire github org and no one seems to be using it.

Reply:
👌 Perhaps I'll leave it like it is for now. If we see problems we can revisit? Or you still think worth making that change?

Reply:
maybe revert to what it was before (with the `const`) so that this PR doesn't have the unnecessary diff?